README.md

```diff
@@ -38,4 +38,4 @@ See [resolve-url-loader](packages/resolve-url-loader/README.md) package.
 
 See [test-my-cli](packages/test-my-cli/README.md) package for a functional programming framework for cli-testing. An unpublished work in progress.
 
-See [resolve-url-loader-filesearch](packages/resolve-url-loader-filesearch/README.md) package for the now defunct file search "magic" from `resolve-url-loader@<3.0.0`. This is currently unpublished. It needs to be adapted as a `join` function to be useful to anyone.
```

Comment:
It's not clear from the diff. But if this is adding an additional line at the end of the file then 👍

Reply:
i guess that is what the online editor of Github inserted on my behalf